src/components/forms/PhoneInput.tsx

```diff
@@ -97,9 +109,20 @@ export const PhoneInput = (props: PhoneInputProps) => {
       if (value === "" || /^[0-9\b]+$/.test(value)) {
         // Persist the internally controlled value
         setNumber(value)
+
         // Format the number as E164 and send it along
+        const phoneNumber = `${
```

Comment:
I think there is a method in the `libphonenumber-js` lib that will output this for you without needing to do string templating like this and ensures it handles all edge cases.

Something like 
```
const phoneNumber = isInternational ? parsePhoneNumber(value, currentCountry.countryCode).number : value
```